resources/sass/components/_card_v2.scss

```diff
@@ -0,0 +1,40 @@
+// Custom style for Card v2.
+// -----------------------------------------------------------------------------
+.card.v2 {
+  .card-header {
+    order: 1;
+  }
+  .card-media {
+    order: 2;
+    flex: 1 0 100%;
+    display: flex;
+    flex-direction: column;
+  }
+  a:not(.card-title a),
+  input,
+  select,
+  a.badge:not(.card-title a),
+  .btn:not(.card-title a) {
+    position: relative;
+  }
```

Comment:
Let's only add the `a` positioning, with a comment on the reason for it.

Reply:
ok, kept only anchors.